conans/client/cache/remote_registry.py

```diff
@@ -132,19 +134,38 @@ def loads(text):
         result = Remotes()
         data = json.loads(text)
         for r in data.get("remotes", []):
-            result._remotes[r["name"]] = Remote(r["name"], r["url"], r["verify_ssl"])
+            disabled = r["disabled"] if "disabled" in r else False
+            result._remotes[r["name"]] = Remote(r["name"], r["url"],
+                                                r["verify_ssl"], disabled)
 
         return result
 
     def dumps(self):
         result = []
         for remote in self._remotes.values():
-            result.append("%s: %s [Verify SSL: %s]" % (remote.name, remote.url, remote.verify_ssl))
+            if remote.disabled:
+                result.append("%s: %s [Verify SSL: %s, Disabled: %s]" %
+                              (remote.name, remote.url, remote.verify_ssl,
+                               remote.disabled))
+            else:
+                result.append("%s: %s [Verify SSL: %s]" %
+                              (remote.name, remote.url, remote.verify_ssl))
```

Comment:
I don't know if there is any reason to not store the disabled value always, it would reduce the number of `ifs`. _(I'm just asking, not proposing a change)_